tests/unit/test_utils.py

```diff
@@ -1263,3 +1264,26 @@ def test_deprecated_message_reads_well():
         "You can find discussion regarding this at "
         "https://github.com/pypa/pip/issues/100000."
     )
+
+
+@pytest.mark.parametrize("setup_py_path", [
+    "setup.py",
+    "/dir/path/setup.py",
+    "../relative/path/setup.py",
+])
+@pytest.mark.parametrize("unbuffered_output", [False, True])
+def test_make_setuptools_shim_args(setup_py_path, unbuffered_output):
+    args = make_setuptools_shim_args(
+        setup_py_path,
+        unbuffered_output=unbuffered_output
+    )
+
+    if unbuffered_output:
+        assert "-u" in args
```

Comment:
This can be simplified to `assert "-u" in args == unbuffered_output` (so no `if` is needed).

Reply:
for some reason `assert "-u" in args == unbuffered_output` fails in py27, i hope this will work

```python
assert ("-u" in args) == unbuffered_output
```